modules/core/src/main/java/org/apache/ignite/internal/processors/failure/FailureProcessor.java

```diff
@@ -124,8 +131,14 @@ public synchronized boolean process(FailureContext failureCtx, FailureHandler hn
         if (this.failureCtx != null) // Node already terminating, no reason to process more errors.
             return false;
 
-        U.error(ignite.log(), "Critical system error detected. Will be handled accordingly to configured handler " +
-            "[hnd=" + hnd + ", failureCtx=" + failureCtx + ']', failureCtx.error());
+        if (isFailureIgnored(failureCtx, hnd)) {
```

Comment:
We can ignore failure type not failure. Please rename method. Also accordingly to code style you should use GG-style getter (no `get` or `is` in method names).